src/radio/radio-controller.ts

```diff
@@ -0,0 +1,233 @@
+import * as dgram from 'dgram';
+import * as net from 'net';
+import {UdpResponse, TcpResponse} from './dataflow';
+
+/**
+ * A default radio timeout, in milliseconds.
+ */
+const RADIO_TIMEOUT = 2500;
```

Comment:
Can this value be overridden? Is 2500ms the right value to use here?

Reply:
Fixed in #48